config/settings.yml

```diff
@@ -10,4 +10,4 @@ google:
   places_api_path: "/maps/api/place/autocomplete/json"
 service_support:
   contact_email_address: becomingateacher@digital.education.gov.uk
-redirect_results_to_c_sharp: true
+redirect_results_to_c_sharp: false
```

Comment:
Too early to remove this?

Reply:
I think we still need to leave it for now. It does need to be set to `true` though 🤦‍♂